solana/ts/scripts/config/devnet/quoter.json

```diff
@@ -28,27 +28,27 @@
   "peerQuotes": {
     "Sepolia": {
       "maxGasDropoffEth": "0",
-      "basePriceUsd": "500000",
-      "nativePriceUsd": "3500000000",
+      "basePriceUsd": "1",
+      "nativePriceUsd": "3865",
```

Comment:
is okay you updated here, but the assistant should take care of this value since sepolia will not be disabled